docs/contribute.md

```diff
@@ -182,12 +241,60 @@ Can include a nice thumbnail.
 ![Notebook_thumbnail](https://airbus.github.io/scikit-decide/maze.png)
 ```
 
-## Unit tests
+### Adding unit tests
 
-Pytest is required to run unit tests. Providing you installed the library in developer mode as described [above](#installing-from-source-in-developer-mode), it should have been already installed by poetry.
+- Whenever adding some code, think to add some tests to the `tests/` folder.
+- Whenever fixing a bug, think to add a test that crashes before fixing the bug and does not afterwards.
 
-From the "scikit-decide" root directory, run unit tests (the "-vv" verbose mode is optional but gives additional details) with:
+Follow [above instructions](#running-unit-tests) to run them with pytest.
 
- ```shell
- poetry run pytest -vv -s tests/solvers/cpp
+## Submitting pull requests
+
+When you think you are ready to merge your modifications into the main repository, you will have to open a pull request (PR).
+We can summarize the process as follows:
+
+- Fork the repository on github.
+- Clone your fork on your computer.
+- Create a branch with a self-explanatory branch, e.g. `xx/add-solver-crazyalgoname` where "xx" stands for your initials,
```

Comment:
External contributors do not need to follow this naming scheme, they can use their master branch if they want.

Reply:
done